includes/class-sensei-analysis-overview-list-table.php

```diff
@@ -804,6 +812,49 @@ public function search_button( $text = '' ) {
 		return $text;
 	}
 
+	/**
+	 * Return an average number of days that students spent to complete the course.
+	 *
+	 * @param WP_Post $item The course to calculate days to completion.
+	 *
+	 * @return float|null Returns an average numbers of days or null if there is not enough data.
+	 */
+	private function get_course_days_to_completion( $item ) {
```

Comment:
Maybe this private function we can test? Using this? https://dev.to/daniel_werner/how-to-use-reflection-to-test-private-and-protected-methods-3339 or add it as separate PR, if it is too much effort? What do you think? It looks like a lot of computation, I also have some PRs where I would want to test private methods.

Reply:
Yes, definitely, that's a good idea. Yesterday, we briefly discussed this approach, @m1r0 suggested his solution with reflection: https://github.com/Automattic/sensei/pull/4863/files#diff-0b0b4b750025f416bbfb18c644dbaf81397ee76e147b50895d57591c5c960401R58-R85